src/prelude.rs

```diff
@@ -29,8 +29,5 @@ pub use dusk_jubjub::JubJubScalar;
 
 /// Collection of errors that the library exposes/uses.
 pub mod plonk_errors {
-    pub use crate::commitment_scheme::kzg10::errors::KZG10Errors;
-    pub use crate::constraint_system::cs_errors::PreProcessingError;
-    pub use crate::fft::fft_errors::FFTErrors;
-    pub use crate::proof_system::proof_system_errors::ProofErrors;
+    pub use crate::error::Error;
```

Comment:
There's no reason to have a module anymore. We should export just like `pub use error::Error`.

Reply:
Makes sense!